templates/pages/admin/assetdefinition/capacities.html.twig

```diff
@@ -41,18 +41,128 @@
 {% set params = params|merge({'formfooter': false}) %}
 
 {% block form_fields %}
+    <style>
+        .red-toggle-switch-bg {
+            --tblr-form-switch-bg: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3e%3ccircle r='3' fill='%23d63939'/%3e%3c/svg%3e") !important;
+        }
+    </style>
+
     <input type="hidden" name="_update_capacities" value="1" />
     {% for capacity in capacities %}
+        {% set label %}
+            <label for="%id%" class="form-label">
+                <span>{{ capacity.getLabel() }}</span>
+            </label>
+        {% endset %}
         {% set field %}
-            <label class="form-check form-switch mt-2">
+            <label class="d-flex align-items-center gap-2 form-check form-switch mt-2">
                 <input type="checkbox"
                     name="capacities[]"
                     value="{{ get_class(capacity)|e('html_attr') }}"
                     class="form-check-input"
                     id="%id%"
+                    data-capacity="{{ get_class(capacity)|e('html_attr') }}"
+                    data-is-used="{{ capacity.isUsed(classname) ? 1 : 0 }}"
                     {{ item.hasCapacityEnabled(capacity) ? 'checked' : '' }} />
+                <i class="ti ti-alert-circle-filled text-red fs-2 d-none"
+                    data-capacity="{{ get_class(capacity)|e('html_attr') }}"
+                    data-bs-toggle="tooltip"
+                    title="{{ capacity.getCapacityUsageDescription(classname) }}"></i>
             </label>
         {% endset %}
-        {{ fields.field('capacities[]', field, capacity.getLabel()) }}
+        {{ fields.field('capacities[]', field, label) }}
     {% endfor %}
+
+    <!-- Modal -->
+    <div class="modal fade" id="capacityWarningModal" tabindex="-1" role="dialog" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <a type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></a>
+                <div class="modal-status bg-danger"></div>
+                <div class="modal-body text-center py-4">
+                    {# SVG Icon corresponding to the ti-alert-triangle icon #}
+                    <svg xmlns="http://www.w3.org/2000/svg" class="icon mb-2 text-danger icon-lg" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"><path stroke="none" d="M0 0h24v24H0z" fill="none"></path><path d="M12 9v4"></path><path d="M10.363 3.591l-8.106 13.534a1.914 1.914 0 0 0 1.636 2.871h16.214a1.914 1.914 0 0 0 1.636 -2.87l-8.106 -13.536a1.914 1.914 0 0 0 -3.274 0z"></path><path d="M12 16h.01"></path></svg>
```

Comment:
Some extra context here for @cedric-anne :

We discussed the UX together today and agreed that SVG icons are better in these case as they scale better to bigger sizes.

----

Should we add the `@tabler/icons` dependency to npm or is it not worth it if we only plan to use it in places where big icons are needed and will keep the css icon as our default everywhere else ?

See https://tabler.io/docs/icons/svg